algorithms/pali.sbl

```diff
@@ -0,0 +1,20 @@
+externals ( stem )
+
+stringescapes {}
+
+stringdef a-   '{U+0101}'  // ā
+stringdef i-   '{U+012B}'  // ī
+stringdef u-   '{U+016B}'  // ū
+stringdef m1   '{U+1E41}'  // ṁ
+stringdef m2   '{U+1E43}'  // ṃ
+
+define stem as (
+  backwards (
+    [substring] delete among (
```

Comment:
While this code is correct, I'd suggest removing the `delete` here and instead adding `(delete)` after line 17 below.  Then it's clearer to a reader less familiar with the Snowball language that the deletion only happens when one of these suffixes matches.  Framing it as an explicit action also more easily allows for different actions for different suffixes if that is useful.